lib/storage/src/issues_subscribers.rs

```diff
@@ -0,0 +1,41 @@
+use std::sync::Arc;
+
+use collection::events::{CollectionDeletedEvent, IndexCreatedEvent, SlowQueryEvent};
+use issues::broker::Subscriber;
+
+use crate::content_manager::toc::TableOfContent;
+use crate::dispatcher::Dispatcher;
+use crate::rbac::Access;
+
+#[allow(dead_code)] // TODO: remove
```

Comment:
still necessary?

Reply:
It will be removed in #4139, we are not accessing `toc` and `access` here yet